client/my-sites/plans-features-main/index.jsx

```diff
@@ -442,7 +410,7 @@ export class PlansFeaturesMain extends Component {
 					compactOnMobile
 				/>
 				<ProductSelector
-					products={ jetpackProducts }
+					products={ JETPACK_PRODUCTS }
 					intervalType={ intervalType }
 					productPriceMatrix={ JETPACK_PRODUCT_PRICE_MATRIX }
```

Comment:
I'm wondering if the the `productPriceMatrix` should be part of the `products` object as well? What do you think?

Reply:
Why not, we could consider that for another PR indeed 👍 Thanks!